src/Payment_link.php

```diff
@@ -16,6 +16,8 @@ class Payment_link extends Entity
      */
     public function create($attributes = array())
     {
+        $attributes = json_encode($attributes);
+        
         $additionHeader['Content-Type'] = 'application/json';
```

Comment:
@neera11 , Request class already have way to add additional headers. why this new function required.

Reply:
@ramth05 
I corrected this, now using Request call addHeader function to add additional header